lib/js/components/SelectionTile/SelectionTile.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div
+		class="selectionTile"
+		:class="[
+			{
+				'-selected': isSelected,
+				'-default': state === SELECTION_TILE_STATE.DEFAULT,
+				'-disabled': state === SELECTION_TILE_STATE.DISABLED,
+				'-loading': state === SELECTION_TILE_STATE.LOADING,
+			},
+		]"
+		@click="updateIsSelected(!isSelected)"
+	>
+		<div class="selectionTile__wrapper">
+			<div class="selectionTile__selectionControl">
+				<radio-button
+					v-if="type === SELECTION_TILE_TYPE.RADIO_BUTTON"
+					:size="RADIO_BUTTON_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+				<checkbox
+					v-else
+					:size="CHECKBOX_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+			</div>
+			<div class="selectionTile__textWrapper">
+				<div class="selectionTile__title">{{ title }}</div>
+
+				<div v-if="supportingText" class="selectionTile__supportingText"
+					>{{ supportingText }}
+				</div>
+			</div>
+		</div>
+		<div
+			v-if="icon || isLoading"
+			class="selectionTile__icon"
+			@click.stop="isLoading ? null : $emit('icon-click')"
+		>
+			<icon
+				:icon="isLoading ? ICONS.FAD_SPINNER_THIRD : icon"
+				:size="ICON_SIZES.X_SMALL"
+				:spinning="state === SELECTION_TILE_STATE.LOADING"
+			/>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography/tokens';
+@import '../../../styles/settings/radiuses';
+
+$selection-tile-min-height: 48px;
+
+.selectionTile {
+	$root: &;
+
+	align-items: center;
+	background-color: $color-neutral-background-weak;
+	border-radius: $radius-s;
+	column-gap: $space-xxs;
+	cursor: pointer;
+	display: flex;
+	justify-content: space-between;
+	min-height: $selection-tile-min-height;
+	outline: 1px solid $color-neutral-border;
+	padding: $space-xs;
+	width: 100%;
+
+	&.-loading {
+		cursor: initial;
+	}
+
+	&__wrapper {
+		align-items: center;
+		column-gap: $space-xxs;
+		display: flex;
+	}
+
+	&__selectionControl {
+		display: flex;
+	}
+
+	&__textWrapper {
+		display: flex;
+		flex-direction: column;
+		row-gap: $space-xxxxs;
+	}
+
+	&__title {
+		@include label-m-default-bold;
+
+		color: $color-neutral-text-heavy;
+	}
+
+	&__supportingText {
+		@include info-m-default-regular;
+
+		color: $color-neutral-text;
+	}
+
+	&__icon {
+		color: $color-neutral-icon;
+		display: flex;
+	}
+
+	&:hover:not(.-loading) {
```

Comment:
should we trigger `hover/focused` state when navigating to Radio using a keybord?

Reply:
but we showing it

Reply:
![Zaznaczenie_146](https://github.com/bethinkpl/design-system/assets/2725894/678f9c9f-7667-45bb-ab08-3198ffae916f)
I see focus only on radio, not entire SelectionTile